rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -6623,16 +6657,13 @@ void opcode_0208(const ScriptArguments& args, const ScriptFloat arg1, const Scri
 	@brief %3d% = random_int_in_ranges %1d% %2d%
 
 	opcode 0209
-	@arg arg1 
-	@arg arg2 
+	@arg min
+	@arg max
 	@arg arg3 
 */
-void opcode_0209(const ScriptArguments& args, const ScriptInt arg1, const ScriptInt arg2, ScriptInt& arg3) {
-	RW_UNIMPLEMENTED_OPCODE(0x0209);
-	RW_UNUSED(arg1);
-	RW_UNUSED(arg2);
-	RW_UNUSED(arg3);
+void opcode_0209(const ScriptArguments& args, const ScriptInt min, const ScriptInt max, ScriptInt& arg3) {
 	RW_UNUSED(args);
+	arg3 = std::rand() % (max - min) + min;
```

Comment:
Beyond the scope: `max - min` may be negative and negative modulo is tricky in C++ iirc.
We should assert that `max > min` return constant if `max == min`